### DISCUSSION TITLE
The Spanish translation is not complete

### DISCUSSION THREAD

**Discussion by jarpingdev** [supermerill/SuperSlicer#2744] [General]
please can you complete the Spanish translation because it is incomplete. Thank you

**Comment by supermerill** [comment]
I don't speak spanish well enough to create a translation.

someone need to complete this translation file https://github.com/supermerill/SuperSlicer/blob/dev/resources/localization/es/TODO.po

**Reply by jarpingdev** [reply]
Prusaslicer is 100% translated, could you use the prusaslicer file and adapt it to superslicer to take advantage of the work done?

**Reply by supermerill** [reply]
already done.
I added /edited many many lines.

**Reply by jarpingdev** [reply]
Great
